app/components/textarea/textarea.rb

```diff
@@ -16,7 +18,7 @@ def call
         class: 'c-textarea',
         data: {
           controller: 'textarea',
-          action: 'input->textarea#resize',
+          action: 'input->textarea#resize'
```

Comment:
I like to have a trailing comma in multiline hash or array literals, because it makes it way more convenient to swap lines if you have them. (Without them, I always forget to add and remove commas… 🙃). Plus, you don’t get extra diff lines when adding lines. But I’ll get used to it… :)